body.tex

```diff
@@ -0,0 +1,74 @@
+
+
+
+\input{intro}
+
+\section{Roles and Reporting}
+
+Tester report issues through Jira, but what other mechanisms will be used?
+
+WHo directs OPS rehersals .. ?
+
+Reports on rehersals .. issues and
+
+Handeling failures - time ines for fix.
+
+
+\begin{note}
+  Not my section but a thought: System Engineering intends to capture commissioning tests through JIRA testing addons/plugins such as Kanoah. We could use some of these to capture functional tests for repeatability between operation rehearsals (they might be overkill for most tests though). --FE
+\end{note}
+
+\begin{note}
+  Note that downstream text refers to ``Software Review Board''. We don't have such an entity so we need to either identify an existing entity or define the constitution of that Board.
+\end{note}
+
+
+% -------------------------------- PASS/FAIL CRITERIA ---------------------------
+
+\input{passfail}
+
+% ------------------------------- CONSTRAINTS AND LIMITATIONS -------------------
+
+\input{constraints}
+
+
+\input{schedule}
+
+% ------------------------------- VALIDATION TOOLS -------------------
+
+\input{tools}
+
+% ------------------------------- UNIT & INTEGRATION TESTS -------------------
+
+\section{Unit and Integration Tests}
+
+\subsection{Approach}
+
+Unit and Integration Tests will be automatically executed through the JUnit test framework. The descriptions of the test below are extracted from the test cases code and documentation.The results of Unit and Integration Test to be included in the Sofwtare Test Report will be generated automatically from the output of the execution of the tests by JUnit. A script will be provided to perform thes processing steps.
+
+Module identification? (module tag in class header? mapping file?)
+
+\subsection{Test Coverage}
+
+Test coverage goal for unit and integration testing. Each class and public method shall have a JUnit test harness that may be labelled according to their purpose (e.g. I/O, individual class/method tests, software integration, data model integration etc.). Nominal and contingency
+tests should be clearly identified.
+
+Interface coverage...
+
+The tool [insert name of unit test coverage tool here] will be used to provide metrics on the code coverage by Unit Tests for \product \ and this metric will be provide in the Test Report.
```

Comment:
we have a plan for adding test coverage to the builds via pytest plus coverage.py. I don't think work has been scheduled for it though.